subsys/caf/modules/ble_adv.c

```diff
@@ -593,7 +597,7 @@ static bool handle_ble_peer_operation_event(const struct ble_peer_operation_even
 		break;
 	}
 
-	return false;
+	return err;
```

Comment:
this must always return false - it does not depend on any error returned by the internal calls